services/parser-database/connector.py

```diff
@@ -56,7 +56,7 @@ def get_articles_that_match_keywords(keywords_list):
         articles_that_match_keyword = {}
         if keyword in keywords_in_memory:
             for article in keywords_in_memory[keyword]:
-                articles_that_match_keyword[str(article["id"])] = article["frequency"]
+                articles_that_match_keyword[str(article["articleNumber"])] = article["frequency"]
```

Comment:
Yes, #68 already got merged, but either way. You got some current conflicting files that must be resolved before merging. I think it would be a good idea to setup a GVC call to resolve them together to reduce the merge friction.